spec/features/tmx_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'TMX' do
```

Comment:
is this supposed to be included? 🙂

Reply:
ahahah whoops not at all.. I will remove thanks